Zend/zend_API.h

```diff
@@ -1681,6 +1684,9 @@ ZEND_API ZEND_COLD void ZEND_FASTCALL zend_argument_value_error(uint32_t arg_num
 #define Z_PARAM_ZVAL(dest) \
 	Z_PARAM_ZVAL_EX(dest, 0, 0)
 
+#define Z_PARAM_ZVAL_OR_NULL(dest) \
+	Z_PARAM_ZVAL_EX(dest, 1, 0)
+
```

Comment:
That seems rather unnecessary as you can check if the type IS_NULL. Or am I missing something?

Reply:
I believe the difference between `Z_PARAM_ZVA(arg)` and `Z_PARAM_ZVAL_OR_NULL(arg)` is that in case of a `null` argument, the former will result in a zval which has a type of `IS_NULL`, while the latter will make `arg` `NULL`.

Reply:
So it's mostly a DX change, gotcha.

Reply:
Can you please commit these separately and adjust existing usages (if any)?

Reply:
Just did it. There was only one usage